packages/commonwealth/shared/utils.ts

```diff
@@ -197,20 +197,19 @@ export function formatAddressShort(
   address: string,
   chain?: string,
   includeEllipsis?: boolean,
-  maxCharLength?: number
+  maxCharLength?: number,
+  prefix?: string
 ) {
   if (!address) return;
   if (chain === 'near') {
     return `@${address}`;
-  } else if (
-    chain === 'straightedge' ||
-    chain === 'cosmoshub' ||
-    chain === 'osmosis' ||
-    chain === 'injective' ||
-    chain === 'injective-testnet' ||
-    chain === 'osmosis-local'
```

Comment:
@alexyoung23j Were these cases just trying to catch long prefixes? Are there places in the app (other than live vote results) I should test?

Reply:
no -- tbh i have no idea why we treat these cases separately. we should probably not.

Reply:
They are all cosmos, so we can handle them whenever we send bech32Prefix into this function.